fluent-package/apt/systemd-test/update-from-v4.sh

```diff
@@ -51,6 +51,20 @@ test $(eval $env_vars && echo $FLUENT_CONF) = "/etc/fluent/td-agent.conf"
 test $(eval $env_vars && echo $FLUENT_PLUGIN) = "/etc/fluent/plugin"
 test $(eval $env_vars && echo $FLUENT_PACKAGE_LOG_FILE) = "/var/log/fluent/td-agent.log"
 
+# Test: logrotate config migration
+test -e /etc/logrotate.d/td-agent
+test -e /var/log/fluent/td-agent.log
+
+sleep 3
+sudo logrotate -f /etc/logrotate.d/td-agent
+
+test -e /var/log/fluent/td-agent.log.1
+sudo cp /var/log/fluent/td-agent.log.1 saved_rotated_logfile
+sudo systemctl stop td-agent
+# Check that SIGUSR1 is sent to Fluentd and Fluentd reopens the logfile
+# not to log to the rotated old file.
+sudo diff -s /var/log/fluent/td-agent.log.1 saved_rotated_logfile
```

Comment:
```suggestion
sudo diff --report-identical-files /var/log/fluent/td-agent.log.1 saved_rotated_logfile
```

Use more meaningful option.

Reply:
Thanks! Fixed!